### ISSUE TITLE
Logs display wrong value when a Blocking Event Hook with response body changed

### ISSUE THREAD

**Issue by yrayegan** [directus/directus#22026]
### Describe the Bug

When an event triggered by flow as a Blocking Event Hook with any Response Body for first time, a log created. Then change the response body to any value and save it, you see all return value converted to last response body value!

### To Reproduce

1. create a collection (blog)
2. create a flow - Event Hook | type: blocking | scope: items.create | collection: blog |  response body: for example "firstLog"
3. create a first blog item
4. check event log - Trigger > option > return:"firstLog"
5. change the flow option now with different response body: now "secondLog"
6. check log again, return:"secondLog"  ???
7. it doesn't matter how many events' log with any response body value created, return value of all created logs converted to last response body value!

https://github.com/directus/directus/assets/79983308/e08c4a6c-77d3-4838-9583-2c0852023135

### Directus Version

v10.10.4

### Hosting Strategy

Self-Hosted (Docker Image)

**Comment by paescuj** [comment]
> Looks like the Trigger Options are just showing the current trigger options not the state of the trigger when the Flow as run 🤔

Yeah, trigger options are not part of revisions, so it always show the current values.
Not sure whether we can / want to take action here. Perhaps show a corresponding notice?

**Comment by br41nslug** [comment]
Well it's displayed as being part of the log for that specific execution. So I think they either don't belong in the logs (as you can see the trigger settings in the flow too) or need to be persisted with the flow logs.

**Comment by licitdev** [comment]
I'm unable to reproduce this issue in `v11.5.1`. Thank you.

![Image](https://github.com/user-attachments/assets/b4da9588-3538-4adc-86d7-7c6b22936ae1)
